aiidalab_widgets_base/structures.py

```diff
@@ -368,16 +369,23 @@ def __init__(self, title="", description="Upload Structure"):
         Supported structure formats
         </a>"""
         )
+        self.output = ipw.HTML("")
```

Comment:
I would say, suggest to use `StatusHTML` instead https://github.com/aiidalab/aiidalab-widgets-base/blob/3897957e33dda90fbbcc0eb8f8b4905bd6d31731/aiidalab_widgets_base/utils/__init__.py#L146

It shows a message for some time, and then it disappears.

Reply:
@yakutovicha thank you. I've changed the code to use the widget.

I must say I am not a big fan of automatically disappearing error messages though. Depending on the size of the message, you might need more time to read _and understand_ it, but if the time is too long, then it might not get cleared before the user takes some other action.

Also, if the message should be transient, it would be nicer to use some kind of hover tooltip, otherwise the UI is jumping up and down. But that's a discussion for another issue I guess. :-)

Reply:
To me, it looked quite okay, tbh.